x/leverage/keeper/token.go

```diff
@@ -197,3 +197,12 @@ func (k Keeper) GetLiquidationIncentive(ctx sdk.Context, denom string) (sdk.Dec,
 
 	return token.LiquidationIncentive, nil
 }
+
+// GetAvailableToBorrow gets the amount available to borrow of a given token.
+func (k Keeper) GetAvailableToBorrow(ctx sdk.Context, denom string) sdk.Int {
+	// Available for borrow = Module Balance - Reserve Amount
+	moduleBalance := k.ModuleBalance(ctx, denom)
+	reserveAmount := k.GetReserveAmount(ctx, denom)
+
+	return moduleBalance.Sub(reserveAmount)
+}
```

Comment:
This can move to `x/leverage/keeper/borrows.go` where similar keeper methods are

Reply:
done at https://github.com/umee-network/umee/pull/379/commits/222e5b97db09336eba8affdf267766c4f6fdf73f